Doppler.Sap.Test/BillingRequestHandlerTest.cs

```diff
@@ -300,5 +312,267 @@ public async Task BillingRequestHandler_ShouldBeNotCreateBillingInSap_WhenQueueH
             Assert.Equal($"The countryCode '' is not supported.", result.SapResponseContent);
             Assert.Equal("Creating Billing Request", result.TaskName);
         }
+
+        [Fact]
+        public async Task BillingRequestHandler_ShouldBeNotUpdatedBillingInSap_WhenQueueHasOneElementButNotExistsInvoiceInSap()
+        {
+            var billingValidations = new List<IBillingValidation>
+            {
+                new BillingForArValidation(Mock.Of<ILogger<BillingForArValidation>>()),
+                new BillingForUsValidation(Mock.Of<ILogger<BillingForUsValidation>>())
+            };
+
+            var sapConfigMock = new Mock<IOptions<SapConfig>>();
+            var timeZoneConfigurations = new TimeZoneConfigurations
+            {
+                InvoicesTimeZone = TimeZoneHelper.GetTimeZoneByOperativeSystem("Argentina Standard Time")
+            };
+            var dateTimeProviderMock = new Mock<IDateTimeProvider>();
+
+            var billingMappers = new List<IBillingMapper>
+            {
+                new BillingForArMapper(Mock.Of<ISapBillingItemsService>(), dateTimeProviderMock.Object, timeZoneConfigurations),
+                new BillingForUsMapper(Mock.Of<ISapBillingItemsService>(), dateTimeProviderMock.Object, timeZoneConfigurations)
+            };
+
+            var httpClientFactoryMock = new Mock<IHttpClientFactory>();
+            var httpMessageHandlerMock = new Mock<HttpMessageHandler>();
+            var httpClient = new HttpClient(httpMessageHandlerMock.Object);
```

Comment:
This variable is not used, you can remove it

Reply:
Done